infra/base-images/base-builder/compile_centipede

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash -eu
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+echo -n "Skipping compilation; using precompiled centipede"
+
+export BIN_DIR="$SRC/centipede/bazel-bin/"
+
+cp "$BIN_DIR/centipede" "$OUT"
+cp "$BIN_DIR/libcentipede_runner.pic.a" "$LIB_FUZZING_ENGINE"
+mkdir "$OUT/corpus"
+mkdir "$OUT/workdir"
+
+export CENTIPEDE_FLAGS=`cat "$SRC/centipede/clang-flags.txt"`
+export COVERAGE_FLAGS="$CENTIPEDE_FLAGS -fsanitize-coverage=trace-loads -ldl -lrt -lpthread"
```

Comment:
Big problem: This is missing trace-pc-guard

Reply:
I am not sure if that is missing in this case:
Running `python infra/helper.py build_fuzzers --engine centipede github_scarecrow --sanitizer coverage` locally produces this stdout:
```
+ /clang/bin/clang++ -O1 -fno-omit-frame-pointer -gline-tables-only -DFUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION -DFUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION -fno-builtin -fsanitize-coverage=trace-pc-guard,pc-table,trace-cmp -O2 -gline-tables-only -fsanitize-coverage=trace-loads -ldl -lrt -lpthread -fprofile-instr-generate -fcoverage-mapping -pthread -Wl,--no-as-needed -Wl,-ldl -Wl,-lm -Wno-unused-command-line-argument -stdlib=libc++ -std=c++11 -Ilib/ /src/github-scarecrow/scarecrow.cc -o /out/scarecrow /usr/lib/libFuzzingEngine.a
```

It includes the flags in `clang-flags.txt` because of [this](https://github.com/google/oss-fuzz/blob/afbe2863895dab2bccdbecd1cd1706b5e09b6358/infra/base-images/base-builder/compile_centipede#L27).

Reply:
Ah ok I assumed :-)